src/Sulu/Component/PHPCR/PathCleanup.php

```diff
@@ -70,6 +74,7 @@ public function cleanup($dirty, $languageCode = null)
         $replacers = $this->replacers['default'];
 
         if (null !== $languageCode) {
+            $languageCode = \str_replace('-', '_', $languageCode);
```

Comment:
@alexander-schranz , should I do this before or after the merge of `replacers` ? :thinking:

Reply:
For BC reasons I would do it after the replacers.

Reply:
thanks, it's fixed